assets/src/components/streetViewButton.tsx

```diff
@@ -35,6 +35,9 @@ const StreetViewButton = ({
     href={streetViewUrl(worldPosition)}
     target="_blank"
     rel="noreferrer"
+    onClick={() => {
+      window.FS?.event("Street view link followed")
```

Comment:
thought(non-blocking): would it be beneficial to save("save" = structured event data 😉 ) the street view link so that we can review later if the links are useful?